packages/components/addon/components/hds/dropdown/list-item.hbs

```diff
@@ -0,0 +1,84 @@
+{{#if (eq this.item "title")}}
+  <li class={{this.classNames}}>
+    {{@text}}
+  </li>
+
+{{else if (eq this.item "generic")}}
+  <li class={{this.classNames}}>
+    {{yield}}
+  </li>
+
+{{else if (eq this.item "copy-item")}}
+  <li class={{this.classNames}}>
+    {{#if @copyItemTitle}}
```

Comment:
since `@title` already exists, I thought that would be a bad idea. The next best thing that came to mind was to add title to the thing that it is, since that felt a little intuitive (tried not to overthink it). Maybe this is something we should ask devs for feedback on? Whether or not the name makes sense in context? WDYT?

Reply:
OK, let's decide once we decide if split or not the `list-item` component in single sub-components.